spec/player_spec.rb

```diff
@@ -0,0 +1,14 @@
+# spec :player_spec.rb
+require './lib/player.rb'
+
+RSpec.describe Player do
+  describe '.change_score' do
```

Comment:
- [ ] **[OPTIONAL] To test initialize or not? [should i test initialize](https://dev.to/leandrogs/should-i-test-initialize)